app/main/forms/auth_forms.py

```diff
@@ -62,6 +62,15 @@ class CreateUserForm(Form):
         ]
     )
 
+    phone_number = StringField(
+        'Phone number', id="input_phone_number",
+        validators=[
+            Regexp("^$|^\\+?([\\d\\s()-]){6,20}$",
+                   message="Phone numbers must be between 6 and 20 characters."
```

Comment:
I don't think this message is quite right. Our pattern requires 6-20 digits (not any old characters), but actually could be up to 21 if they have an optional `+` at the start.  We don't want to get into those details in the message though.

How about `You must enter a valid phone number.`?  But probably best to check with @superroz or Sue for the ideal wording.